### DISCUSSION TITLE
Issue with Rendering Emojis in GitHub Workflow Comments Using github-script@v6

### DISCUSSION THREAD

**Discussion by alexisaleksandra** [community/community#118420] [Enterprise]
Hi,

I’ve encountered an issue on GitHub Enterprise Server 3.12.1 where emojis are not rendered correctly when using the github-script action (version 6) in GitHub workflows.

Environment:

**GitHub Enterprise Server 3.12.1
github-script@v6**
Issue Description: When attempting to add comments to issues via a GitHub workflow using the github-script action, the emojis are displayed as their literal text codes rather than the corresponding graphical icons.

Steps to Reproduce:

Create a GitHub workflow that uses github-script@v6.
Add a step to create an issue comment with emojis using the following script:
```
- name: Comment with Emoji
  uses: actions/github-script@v6
  with:
    script: |
      const body = `:x: Please don't malform yaml structure.`
      github.rest.issues.createComment({
        issue_number: context.issue.number,
        owner: context.repo.owner,
        repo: context.repo.repo,
        body
      })
```
Expected Behavior: The comment on the GitHub issue should display the red “x” emoji.

Actual Behavior: The comment is posted with the literal string “:x:” instead of the emoji. The same issue occurs with other emojis, such as “white_check_mark”.

Additional Information:

This issue seems to have arisen after updating to GitHub Enterprise Server 3.12.1.
The expected behavior was previously observed with older versions of the github-script action.

Thank you for your assistance.

**Comment by queenofcorgis** [comment]
Hi there @alexisaleksandra 👋🏼 , 

Appreciate the patience!

If you wouldn't mind sharing what version you upgraded _from_ that would be helpful for us to know. 

Some additional troubleshooting: 
* You mentioned that it shows `":x:"` in the issue, and not the emoji. Are you able to collect and share the raw text from the issue? AKA go to the issue and click the `...` so that you can edit it and collect the text that is there.
* If you enter `:x:` manually into an issue/comment how is it rendered?

**Comment by wouterklouwen-youview** [comment]
Hi, We recently updated from GHES 3.11.3 to 3.12.1 and found the same issue where emoticons that are at the begining of a line in a GitHub Check Run's description does not render. If they are in the middle or at the end, they render as before. When typing them in manually in a comment, they appear.
